assets/_scss/components/_footer.scss

```diff
@@ -0,0 +1,298 @@
+        right: 2.5rem;
+      }
+    }
+  }
+}
+
+.usa-footer-medium {
+  
+  .usa-footer-primary-section {
+    padding: 0;
+    
+    @media (min-width: $tablet-up) {
+      padding: {
+        top: 1rem;
+        bottom: 1rem;
+      }
+    }
+  }
+
+  .usa-footer-nav ul {
+    
+    @media (min-width: $tablet-up) {
+      align-items: center;
+      display: flex;  
+    }
+  }  
+}
+
+.usa-footer-slim {
+
+  .usa-footer-nav a {
+    display: block;
+  }
+
+  .usa-footer-primary-section {
+    padding-bottom: 2rem;
+    
+    @media (min-width: $tablet-up) {
+      padding: {
+        top: 1rem;
+        bottom: 1rem;
+      }
+
+      .grid-box {
+        align-items: center;
+        display: flex;     
+      }      
+    }
+  }
+}
+
+.usa-footer-slim li,
+.usa-footer-medium li {
+  
+  &:last-child {
+    border-bottom: 1px solid $color-base-link;
+
+    @media (min-width: $tablet-up) {
+      border-bottom: none;
+    }
+  }
+}
+
+.usa-footer-nav {
+  
+  li {
+
+    @media (min-width: $tablet-up) {
+      margin: 1rem 0;
+    }
+  }
+}
+
+.usa-sign_up-block {
+  padding: {
+    bottom: 2rem;
+    left: 2.5rem;
+    right: 2.5rem;
+  }
+
+  button {
+    float: none;
+    margin: {
+      right: 0;
+      top: 1.5rem;
+    }
+  }
+
+  input {
+    width: 100%;
+  }
+
+  @media (min-width: $tablet-up) {
+    float: right;
+    padding: 0;
+  }  
+}
+
+.usa-footer-secondary-section {
+  background-color: $color-grey-300;
+  padding: {
+    top: 3rem;
+    bottom: 3rem;
+  }
+}
+
+.usa-footer-big-secondary-section {
+  
+  @media (min-width: $tablet-up) {
+    padding-top: 5rem;
+  }  
+}
+
+.usa-footer-contact-links {
+
+  @media (min-width: $tablet-up) {
+    text-align: right;
+  }
+}
+
+.usa-footer-big {
+
+  .usa-footer-primary-section {
+
+    @media (min-width: $tablet-up) {
+      padding: {
+        top: 3rem;
+        bottom: 4rem;
+      }      
+    }
+  }
+
+  ul {
+
+    padding-bottom: 2.5rem;
+
+    &:last-child {
+      border-bottom: 1px solid $color-base-link;
+
+      @media (min-width: $tablet-up) {
+        border-bottom: none;
+      }
+    }
+
+    @media (min-width: $tablet-up) {
+      padding-bottom: 0;
+    }    
+
+    .usa-footer-primary-link {
+      background: url('../img/arrow-down.png') no-repeat;
+      background-position: 1.5rem;
+      padding-left: 3.5rem;
+
+      @media (min-width: $tablet-up) {
+        background: none;
+        padding-bottom: 1rem;
+        padding-left: 0;
+      }
+    }
+
+    &.hidden {
+
+      padding-bottom: 0;
+
+      .usa-footer-primary-link { 
+        margin: 0; 
+        background: url('../img/arrow-right.png') no-repeat;
+        background-position: 1.5rem;
+
+        @media (min-width: $tablet-up) {
+          background: none;
+          padding-left: 0;
+        }        
+      }
+        
+      li { display: none; }
+      
+    }
+  }
+}
+
+.usa-footer-topic, 
+.usa-sign_up-header {
+  margin: 0;
+  padding: 2rem 0;
+}
+
+img.usa-footer-logo-img {
```

Comment:
Why do you need the `img` part of the selector? Wouldn't `.usa-footer-logo-img` be enough?